Remove old versions of the duplicated dependencies.
Remove wasm dependencies.
Tweak versions to what is packaged in Debian.

diff --git a/Cargo.toml b/Cargo.toml
index fcbceb9..f279b89 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -93,10 +93,6 @@ version = "0.11.5"
 version = "1.0"
 features = ["preserve_order"]
 
-[dependencies.serde_with]
-version = "1.3.1"
-optional = true
-
 [dependencies.serde_with-3]
 version = "3.1.0"
 optional = true
@@ -118,14 +114,14 @@ features = [
     "v4",
 ]
 
-[dependencies.uuid-0_8]
-version = "0.8.1"
-features = [
-    "serde",
-    "v4",
-]
-optional = true
-package = "uuid"
+#[dependencies.uuid-0_8]
+#version = "0.8.1"
+#features = [
+#    "serde",
+#    "v4",
+#]
+#optional = true
+#package = "uuid"
 
 [dev-dependencies.assert_matches]
 version = "1.2"
@@ -143,7 +139,7 @@ default-features = false
 version = "0.3.0"
 
 [dev-dependencies.pretty_assertions]
-version = "0.6.1"
+version = "1"
 
 [dev-dependencies.proptest]
 version = "1.0.0"
@@ -157,9 +153,9 @@ default = []
 time-0_3 = []
 uuid-1 = []
 
-[target."cfg(all(target_arch = \"wasm32\", target_os = \"unknown\"))".dependencies.js-sys]
-version = "0.3"
-
-[target."cfg(all(target_arch = \"wasm32\", target_os = \"unknown\"))".dev-dependencies.getrandom]
-version = "0.2"
-features = ["js"]
+#[target."cfg(all(target_arch = \"wasm32\", target_os = \"unknown\"))".dependencies.js-sys]
+#version = "0.3"
+#
+#[target."cfg(all(target_arch = \"wasm32\", target_os = \"unknown\"))".dev-dependencies.getrandom]
+#version = "0.2"
+#features = ["js"]
